File: redcap_v15.7.6/API/examples/perl/export_logging.pl
#!/usr/bin/env perl

use strict;
use warnings;
use LWP::Curl;

my %config = do 'config.pl';

my $fields = {
    token     => $config{api_token},
    content   => 'log',
    format    => 'json',
    logtype => '',
    user => '',
    record => '',
    beginTime => '10/06/2020 17:37',
    endTime => ''
};

my $ch = LWP::Curl->new(auto_encode => 0);
my $content = $ch->post($config{api_url}, $fields, $config{referer});

print $content;

File: redcap_v15.7.6/API/examples/python/export_logging.py
#!/usr/bin/env python

from config import config
import requests

fields = {
    'token': config['api_token'],
    'content': 'log',
    'format': 'json',
    'logtype': '',
    'user': '',
    'record': '',
    'beginTime': '10/06/2020 17:37',
    'endTime': '',
}

r = requests.post(config['api_url'],data=fields)
print('HTTP Status: ' + str(r.status_code))
print(r.text)